### PR TITLE
[MINI-3314] Cache was kept in preview mode

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
There was a double check to keep cache while in preview mode and only one was removed

## Links
[MINI-3314](https://jira.rakuten-it.com/jira/secure/RapidBoard.jspa?rapidView=21325&projectKey=MINI&view=detail&selectedIssue=MINI-3314)

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
👍👍

File: MiniApp/Classes/core/Downloader/MiniAppDownloader.swift [hunk 1]
```diff
@@ -29,9 +29,7 @@ class MiniAppDownloader {
     }
 
     fileprivate func cleanApp(_ appId: String, for version: String) {
-        if self.miniAppClient.environment.isPreviewMode == false {
-            self.miniAppStorage.cleanVersions(for: appId, differentFrom: version, status: self.miniAppStatus)
-        }
+        miniAppStorage.cleanVersions(for: appId, differentFrom: version, status: miniAppStatus)
```
Comment (rleojoseph) [inline MiniApp/Classes/core/Downloader/MiniAppDownloader.swift:32]: So we are going to download the Mini app every time irrespective of the Preview or Published mode?

**Review by rleojoseph — COMMENTED** [review]
Looks ok to me, just waiting for the removal of UI testcases.

File: MiniApp/Classes/core/Downloader/MiniAppDownloader.swift (hunk 1 above)
Comment (Climbatize) [reply MiniApp/Classes/core/Downloader/MiniAppDownloader.swift:32]: no, the check is done line 37 @rleojoseph ;)

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**